firmware/Core/Src/unit_tests/test_ant_commands.c

```diff
@@ -0,0 +1,21 @@
+#include "unit_tests/unit_test_helpers.h"
+#include "unit_tests/test_assorted_prototypes.h"
+#include<antenna_deploy_drivers/ant_commands.h>
+
+uint8_t TEST_EXEC__ANT_CMD_measure_temp() {
```

Comment:
Can you please add a link to the GitHub issue in here that has the derivations for these unit tests?